tests/lib/lore_test.sh

```diff
@@ -440,4 +441,114 @@ function test_save_new_lore_config()
   assert_equals_helper 'Wrong lore.config contents' "$LINENO" "$expected" "$output"
 }
 
+function test_compose_lore_query_url_with_verification()
```

Comment:
I suggest you split these tests by cases. For example, move the invalid option to its own test and the good cases as well.